source/AlgebraicMethods/Log.cpp

```diff
@@ -271,8 +271,7 @@ void Log::FormatXMLString() {
   const char *rules[] = {"$", "",     "\\{",     "{",          "\\}",
                          "}", "\\ni", "&#8715;", "\\emptyset", "&#8709;"};

```

Comment:
Outside the diff of this PR itself, but this is typoed as “emtpyset” in the leading comment.